### PR TITLE
Retrain ML Model How-To

### PR DESCRIPTION
**Pull request by luisquintanilla**
Retrain ML Model How-To

[Internal Review Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/retrain-model-ml-net?branch=pr-en-us-11993)

#AB1442463

### CODE REVIEW

File: docs/machine-learning/how-to-guides/retrain-model-ml-net.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+// Load data prepration pipeline
+ITransformer dataPrepPipeline = mlContext.Model.Load("data_preparation_pipeline.zip", out dataPrepPipelineSchema);
+
+// Load trained model
+ITransformer trainedModel = mlContext.Model.Load("ogd_model.zip", out modelSchema);
+```
+
+## Extract pre-trained model parameters
+
+Once the model is loaded, extract the learned model parameters by accessing the [`Model`](xref:Microsoft.ML.Data.PredictionTransformerBase`1.Model*) property of the pre-trained model. The pre-trained model was trained using the linear regression model [`OnlineGradientDescentTrainer`](xref:Microsoft.ML.Trainers.OnlineGradientDescentTrainer) which creates a[`RegressionPredictionTransformer`](xref:Microsoft.ML.Data.RegressionPredictionTransformer`1) that outputs [`LinearRegressionModelParameters`](xref:Microsoft.ML.Trainers.LinearRegressionModelParameters). These linear regression model parameters contain the learned  bias and weights or coefficients of the model. These values will be used as a starting point for the new re-trained model.
```
Comment (JRAlexander) [inline docs/machine-learning/how-to-guides/retrain-model-ml-net.md]: learned  bias <- remove extra space.

**Review by JRAlexander — APPROVED** [review]
LGTM. One small nit.

**Review by natke — COMMENTED** [review]
Looks great! Need to add this article to the TOC.

Also, can we change the title to Retrain a model

File: docs/machine-learning/how-to-guides/retrain-model-ml-net.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/retrain-model-ml-net.md]: #resolved

**Pull request merged**